BUNDLED_NODE_VERSION

```diff
@@ -1 +1 @@
-v10.20.1
+v12.0.0
```

Comment:
For upgrading to Electron 7, we should update this to at least 12.13.1.
NSFW needs this for compiling: https://github.com/atom-ide-community/atom/pull/75